src/main/webapp/scripts/tutorial-scripts/facebook-template.js

```diff
@@ -0,0 +1,76 @@
+import {tutorialTemplate} from './tutorial-template.js';
+
+const facebookData = {
+  toolName: 'Facebook',
+  toolDescription: 'Social Media Tool',
+  toolLink: 'https://www.facebook.com/',
+  usefulLinks: [
+    {linkTitle: 'Getting started with Facebook',
```

Comment:
nitpick: The `linkTitle` should be on a new line and indented. This looks like an issue in a bunch of places below and in the other similar files (e.g. I see it happening with `url` too). Should be like:

```
usefulLinks: [
  {
    linkTitle: 'Getting started with Facebook',
    links: [
      {
        url: 'https://www.wikihow.com/Use-Facebook',
        linkName: 'How to Use Facebook',
      },
    ],
  },
  {
    ...
  },
],
```

(I may be wrong about this.. if the linter told you to format it this way, trust that over me!)

Reply:
Linter didn't throw any errors so I changed all the indentations accordingly.